Package.swift

```diff
@@ -157,6 +157,14 @@ var targets: [Target] = [
             "StencilSwiftKit",
         ]
     ),
+    .target(
+        name: "TuistGeneratorTesting",
+        dependencies: [
+            "TuistGenerator",
+            swiftToolsSupportDependency,
+        ],
+        linkerSettings: [.linkedFramework("XCTest")]
+    ),
```

Comment:
I needed this to be able to run the rests for TuistGenerator, see LinkGenerator. I do feel like I'm missing something here so would appreciate clarification. 

[Testing Strategy](https://docs.tuist.io/contributors/testing-strategy) relies on fourier which seems to have been superseded in the project?

Reply:
The testing strategy document is outdated. We are working on a new documentation towards the release of Tuist 4, which is coming in January.